app/next-client-app/components/homepage/hero.tsx

```diff
@@ -0,0 +1,31 @@
+import { AnimatedBeamMapping } from "@/components/AnimatedBeamMapping";
+import AnimatedGridPattern from "@/components/magicui/animated-grid-pattern";
+import { cn } from "@/lib/utils";
+
+export default function Hero() {
+  return (
+    <>
+      {/* Hero section */}
+      <div className="flex-col overflow-hidden lg:flex lg:flex-row lg:justify-between items-center pt-10">
+        <div className="flex flex-col lg:space-y-5 space-y-3 lg:w-3/5">
+          <div className="lg:text-5xl text-3xl gap-5 text-center lg:text-left">
+            <h1 className="text-orange-600">OMOP Transforming </h1>
+            <h1> Made Easy</h1>
+          </div>
+          <h3 className="text-gray-600 dark:text-gray-200 sm:text-lg text-md mt-2 text-justify lg:text-left">
+            Carrot utilizes scan reports' metadata to generate JSON mapping
```

Comment:
`Carrot automates vocabulary mapping to OMOP, enabling rule reuse across datasets and flexible manual mapping, streamlining data integration for faster research.`